pkgs/development/python-modules/docutils/default.nix

```diff
@@ -8,15 +8,15 @@
 
 buildPythonPackage rec {
   pname = "docutils";
-  version = "0.19";
+  version = "0.20.1";
 
   disabled = pythonOlder "3.7";
 
   format = "setuptools";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-M5laZ1PDC39Xf+v8LFBBH+xqrH9//rfEz+WZEHLc+eY=";
+    sha256 = "8IpOJ2w6FYOobc4+NKuj/gTQK7ot1R7RYQYkToqSPjs=";
```

Comment:
```suggestion
    hash = "sha256-8IpOJ2w6FYOobc4+NKuj/gTQK7ot1R7RYQYkToqSPjs=";
```

Reply:
I don’t have a strong feeling, but why is more characters preferred to less?

Reply:
It is more generalizable and future-proof.


If e.g. the default hash changes to SHA512 (or any other algorithm), `hash = <algo>-<value>;` generates less noise than `<algo> = <value>;`.